packages/platform-core/API.md

```diff
@@ -136,13 +136,13 @@ export type LocalConfigurationFileName = 'usage_data_preferences.json';
 // @public
 class LogLevelConverter {
     // (undocumented)
-    toApplicationLogLevel: (logLevel: LogLevel | undefined) => ApplicationLogLevel | undefined;
+    toCDKApplicationLogLevel: (logLevel: LogLevel | undefined) => ApplicationLogLevel | undefined;

```

Comment:
I'd still suggest to include `lambda` in there, as other AWS services may have different log level names or types.
```suggestion
    toCDKLambdaApplicationLogLevel: (logLevel: LogLevel | undefined) => CDKLambdaApplicationLogLevel | undefined;
```

Reply:
I can change the `toCDKApplicationLogLevel`.

But return type is already `import { ApplicationLogLevel } from 'aws-cdk-lib/aws-lambda';`
Even if we alias this for our purposes, customer still needs to import from cdk lib.